docs/platform/what-is-tizen/profiles/iot.md

```diff
@@ -0,0 +1,118 @@
+
+
+## Types of Building Block Presets
+
+### IoT Core
+- The minimum set of bootable modules that make up the Tizen platform image for IoT devices.
+- The preset contains core components that are required for a basic system image to boot.
+
+### Headless
+- The set of modules that make up the Tizen platform image for headless IoT devices without a display.
+- The preset contains IoT core presets with other components specifically required for headless image.
+- Example of specific components for headless includes Audio moduldes, Bluetooth modules, and etc.
+
+### Headed
+- The set of modules that make up the Tizen platform image for headed IoT devices with a display.
+- The preset contains IoT core presets with other components specifically required for headed image.
+- Example of specific components for headed includes Notification API, graphic modules, and etc.
+
+
+## MIC Imager Creator
+
+MIC is an image creator that is used to create images for Tizen. 
+
+The tool offers three major functions as follows:
+- creating an image in a specific format
+- chrooting into an image
+- coverting an image to another format
+
+For more information about the tool and its functions, refer to [MIC Image Creator](https://docs.tizen.org/platform/reference/mic/mic-overview/).
+
+In this document, we will focus on creating the image using MIC.
+
+To create a custom Tizen IoT image, 
+
+0) Check prerequisites
+    - OS: Ubuntu 14.04 LTS or higher
+    - Git Build System(GBS) and Image Creator(MIC) for Tizen developers
+        ```shell
+        $ echo "deb [trusted=yes] http://download.tizen.org/tools/latest-release/Ubuntu_16.04/ /" | sudo tee /etc/apt/sources.list.d/tizen.list
+        $ sudo apt-get update
+        $ sudo apt-get install gbs mic
+        ```
+
+1) Download a kistart(.ks) file from the [Tizen download site](http://download.tizen.org/snapshots/tizen/unified/). 
+    - You may choose the file for the binary that is most similar to the custom image you are trying to create.
+
+2) Edit the kickstart file in accordance with the requirements of the custom image. 
+    - You can add the name of the required building blocks under the %pakckages section of the kickstart file.
+        ```shell
+        %packages
+
+        # @ IoT Headless Base
+        building-blocks-root-Preset_iot_core
+        # @ IoT Adaptation RPI
+        building-blocks-sub1-Preset_img_headless-rpi3
+        # Others
+        # camera blocks  <-- New building block to use camera feature added
+        building-blocks-sub2-Preset_boards-RPI3-Camera
+        
+        %end
+        ```
+    - To find the name of a specific building block, refer to the [mapping-bb-rs.xml](https://git.tizen.org/cgit/tools/building-blocks/plain/mapping-bb-rs.xml?h=tizen) file in the [Tizen building block repository](https://git.tizen.org/cgit/tools/building-blocks).
+
+3) Create image with the gbs command.
```

Comment:
Suggestion:

3. Create image with the git-build-systemm (gbs) command:

Reply:
done :)